### ISSUE TITLE
Cannot inline bytecode due to JVM target version

### ISSUE THREAD

**Issue by daFant** [AlexanderPrendota/kotlin-compiler-server#576]
Hi,

first of all: thank you for this amazing & fun project! :-)

I unfortunately ran into some problems using custom dependencies for a Kotlin DSL which is using a lot of inline functions. The error message I keep getting is:

`Cannot inline bytecode built with JVM target 11 into bytecode that is being built with JVM target 1.8. Please specify proper '-jvm-target' option`

Its true that the dependencies are built with JVM target 11, but I also configured the jvmTarget for the kotlin-compiler-server to be 11:

```
tasks.withType<KotlinCompile> {
    kotlinOptions {
        freeCompilerArgs = listOf("-Xjsr305=strict", "-Xskip-metadata-version-check")
        jvmTarget = "11"
    }
// ...
}
```

And I also defined a Java toolchain for the server, executors, common and indexation module:
```
java {
  toolchain {
    languageVersion.set(JavaLanguageVersion.of(11))
  }
}
```

The Java versions of the compiled class files are OK (55), but I'm always getting the error message above.

Thanks in advance for your help.
Regards

**Comment by AlexanderPrendota** [comment]
Hi, thanks. 

I'm going to update the project to java 11 or more. I hope it helps you, am I right?

**Comment by daFant** [comment]
Yes, I think that might help! Thanks :-)

**Comment by daFant** [comment]
Unfortunately the update didn't work for me and I'm still getting the same error.

In my opinion you can leave this issue closed as long as #579 is open, as it should be the same issue.
